saadaal-flood-forecaster/flood_forecaster/utils/configuration.py

```diff
@@ -0,0 +1,67 @@
+import configparser
+from dataclasses import dataclass
+import json
+import os
+from typing import List
+from configparser import ConfigParser
+
+
+DEFAULT_CONFIG_FILE_PATH = os.path.dirname(os.path.realpath(__file__)) + "/../../../config/config.ini"
+
+
+class Config:
+    def __init__(self, config_file_path: str) -> None:
+        self._config: ConfigParser = self._load_config(config_file_path)
+
+    def load_database_config(self):
+        return dict(self._config.items("database"))
+
+    def load_openmeteo_config(self):
+        return dict(self._config.items("openmeteo"))
+
+    @staticmethod
+    def _load_config(config_file_path: str) -> configparser.ConfigParser:
+        """
+        Load configuration from the given file path.
+
+        :param config_file_path: Path to the configuration file
+        :return: ConfigParser object
+        """
+        if not os.path.exists(config_file_path):
+            raise FileNotFoundError(f"Config file '{config_file_path}' not found.")
+
+        config = configparser.ConfigParser()
+        config.read(config_file_path)
+        return config
+
+
+@dataclass
+class StationMapping:
```

Comment:
Nice but already inside `openmeteo/station.py`. we need to unify all this asap.
Even Maxime already started to refactor it, moving station.py to data_model, which for me makes sense since it's a concept from the application. Like weather and river_level